Curricular Models/GenEvo/GenEvo 4 Competition.nlogo

```diff
@@ -48,28 +48,28 @@ to generate-cells
     set shape "ecoli"
 
     ifelse choose-models? [
-      user-message "Select a Genetic Switch model to load..."
+      user-message "Select a Genetic Switch model to create..."
```

Comment:
@qiemem, @bainco: Not sure what to suggest instead, but that turn of phrase is awkward. Maybe "Select a Genetic Switch model to use as a child model..."? Or "from which to create a child model"? They're both a mouthful, though.